src/modules/validator.js

```diff
@@ -62,14 +62,14 @@ class Validator {
 
     async validateRRSignature(rr, expectedRRPublicKey = null) {
         this.chluIpfs.logger.debug('Validating RR signature');
-        const pubKeyMultihash = this.keyLocationToKeyMultihash(rr.key_location);
-        const isExpectedKey = expectedRRPublicKey === null || expectedRRPublicKey === pubKeyMultihash;
+        const didId = rr.key_location
```

Comment:
So you are saving the didId in the key_location. Didn't we discuss that we should add a new attribute in the review record called "did_id"?

Reply:
Yeah, it's in a list of things that need to be done next in the original PR description. I'll get this done today